exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -361,4 +361,11 @@
 
   String DYNAMIC_UDF_SUPPORT_ENABLED = "exec.udf.enable_dynamic_support";
   BooleanValidator DYNAMIC_UDF_SUPPORT_ENABLED_VALIDATOR = new BooleanValidator(DYNAMIC_UDF_SUPPORT_ENABLED, true, true);
+
+  /**
+   * Option representing Drill cluster version
+   */
+  String CLUSTER_VERSION = "drill.exec.cluster.version";
```

Comment:
What is a cluster version? Is this a well-defined concept? If there is such a thing, isn't it the version of all Drillbits in the cluster? It would not be a config parameter, would it?